scale/job/models.py

```diff
@@ -2643,6 +2648,11 @@ def edit_job_type_v6(self, job_type_id, manifest=None, docker_image=None, icon_c
             job_type.author_url = manifest.get_maintainer()['url']
             job_type.save()
```

Comment:
After the job type save here, we need a call to `JobTypeRevision.objects.create_job_type_revision(job_type)` just like during job type creation so there is a new revision for the manifest changes. The call must be after the job type save because the call assumes that `manifest` and `revision_num` have been updated.

Reply:
Never mind, it's there. :)